bin/webpack

```diff
@@ -25,9 +25,10 @@ end
 #   If nodejs is compiled with openssl 1.1, then this flag is not available
 #     and will fail if we hardcode it into NODE_OPTIONS
 #
-# This can be removed once we upgrade webpack as then we don't need the
+# This can be removed once we upgrade from webpack 4.0 to 5.0 as then we won't need the
 # flag anymore.
-has_legacy_openssl = system('node --help | grep -qs "\\--openssl-legacy-provider"')
+`node --openssl-legacy-provider --openssl-legacy-provider -v`
```

Comment:
What is this change for?

Reply:
Not that the check here is because not all compilations of node have this option. If you try to call node when the option doesn't exist it blows up.

Reply:
sorry, added to slack but not here. Will copy here since that is where it belongs.

The old `node --help | grep` for some reason was failing sporadically locally.

I just put that in so it will fail if is not available and we are looking at `$?`

I'll fix the typo for double typing the `--openssl-legacy-provider`